### DISCUSSION TITLE
Add «all activity» to mobile apps

### DISCUSSION THREAD

**Discussion by gko** [community/community#3042] [Mobile]
Right now when I open app and I go to `notifications` what I see mostly are issues/comments. Generally speaking, stuff that requires some action from me. 

But on the main page of `github` when I'm logged in the first thing I see are the activities of the people that I follow and/or stars/forks of my repositories.

I think that having this on a front page is great because this way I can find out about latest trends and actually get inspired to work on issues and stuff that requires action.

**Comment by circa10a** [comment]
This is what is preventing me from using the GitHub mobile app. I rely on fasthub for this functionality

**Comment by finalchange23** [comment]
> Right now when I open app and I go to `notifications` what I see mostly are issues/comments. Generally speaking, stuff that requires some action from me. 
> 
> 
> 
> But on the main page of `github` when I'm logged in the first thing I see are the activities of the people that I follow and/or stars/forks of my repositories.
> 
> 
> 
> I think that having this on a front page is great because this way I can find out about latest trends and actually get inspired to work on issues and stuff that requires action.

**Comment by Anas198900** [comment]
> Right now when I open app and I go to `notifications` what I see mostly are issues/comments. Generally speaking, stuff that requires some action from me. 
> 
> 
> 
> But on the main page of `github` when I'm logged in the first thing I see are the activities of the people that I follow and/or stars/forks of my repositories.
> 
> 
> 
> I think that having this on a front page is great because this way I can find out about latest trends and actually get inspired to work on issues and stuff that requires action.

**Comment by Anas198900** [comment]
> This is what is preventing me from using the GitHub mobile app. I rely on fasthub for this functionality

**Comment by Anas198900** [comment]
> > This is what is preventing me from using the GitHub mobile app. I rely on fasthub for this functionality
> 
>
